Test fails, and it seems to be due to changes in Alpine's musl vs the hardcoded
list of locales in the test.

https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/main/musl-locales/add-all-available-locales.patch


diff --git a/tests/test-compose.c b/tests/test-compose.c
index 89a5b101..b646827a 100644
--- a/tests/test-compose.c
+++ b/tests/test-compose.c
@@ -898,7 +898,6 @@ main (int argc, char **argv)
 	g_test_add ("/AppStream/Compose/OptipngNotfound", Fixture, NULL, setup, test_compose_optipng_not_found, teardown);
 	g_test_add_func ("/AppStream/Compose/Utils", test_utils);
 	g_test_add_func ("/AppStream/Compose/IssueTagSanity", test_compose_issue_tag_sanity);
-	g_test_add_func ("/AppStream/Compose/FontInfo", test_read_fontinfo);
 	g_test_add_func ("/AppStream/Compose/Image", test_image_transform);
 	g_test_add_func ("/AppStream/Compose/Canvas", test_canvas);
 	g_test_add_func ("/AppStream/Compose/Hints", test_compose_hints);
